mediorum/ddl/cid_lookup.sql

```diff
@@ -7,25 +7,6 @@ create table if not exists cid_lookup (
     "host" text
 );
 
--- creator-node creates the Files table, but we need to create it here in case we're running via audius-compose without CNs
```

Comment:
I think we need this because audius-compose by default runs with CNs. the rest of the cid lookup code depends on this Files table existing, which wouldn't be the case locally without CNs running. could we add it back or do you know if this already works for some reason?

Reply:
oh yeah will re-add